### PR TITLE
 Improve CLI feedback and documentation for checksum checking and repair

### PR DESCRIPTION
**Pull request by kjsanger**
Improve CLI feedback and documentation for checksum checking and repair

Add --colour and --json CLI options.
Add counting of items, errors, repairs.
Add summary information of errors to STDERR.

(These 2 commits should be squashed)

### CODE REVIEW

**Comment by mksanger** [comment]
It could probably do with a test to check that it returns the correct tuple

**Comment by kjsanger** [comment]
> It could probably do with a test to check that it returns the correct tuple

Good point. I've added some to cover the basics (not all scenarios e.g. not completely testing the output strings)

File: tests/test_utilities.py [hunk 1]
```diff
@@ -17,18 +17,109 @@
...
-from npg_irods.utilities import copy
+from npg_irods.metadata.common import ensure_common_metadata
+from npg_irods.utilities import check_checksums, copy, repair_checksums
 
 
 @m.describe("Utilities")
 class TestUtilities:
+    @m.context("When data object checksums are checked")
+    @m.context("When all of the data objects have checksum metadata")
+    @m.it("Counts repairs correctly")
```
Comment (mksanger) [inline tests/test_utilities.py:37]:
```suggestion
    @m.it("Counts correct checksums correctly")
```

**Comment by kjsanger** [comment]
Can we squash all these commits on merge, please?

**Pull request merged**